Bugzilla/Report/Ping/Simple.pm

```diff
@@ -49,8 +49,7 @@ sub _build_validator {
     creation_ts      => joi->string->required,
   });
 
-  return JSON::Validator->new(
-    schema => Mojo::JSON::Pointer->new($schema->compile));
+  return JSON::Validator->new(schema => $schema);
```

Comment:
This is an unrelated change (probably due to the `cpan.snapshot` update) and should be in another patch IMO

Reply:
Yes this is due to the newer version of JSON::Validator due to the new bmo-slim-perl image built with the latest CPAN dependencies. I can pin the version of JSON::Validator to an older version and build a new image. The reason for the new image in the first place was due to needing new version of the mysql perl bindings to support the move the 8.x.

Reply:
Ah, that's fair.
Would you mind splitting the bumping of the new `bmo-slim-perl` image (plus fixing deprecated usages, such as `JSON::Validator` here) from the MSYQL8 changes?

That way, if there's risk of breakage, it may be easier to track down which change caused it (changed system + dependencies versus the signficant MySQL change).